libs/blocks/global-navigation/global-navigation.css

```diff
@@ -320,14 +320,23 @@ header.global-navigation {
   flex-shrink: 0;
 }
 
+.feds-breadcrumbs li:last-child:not(:first-child) {
+  padding-right: 12px;
+}
+
+[dir = "rtl"] .feds-breadcrumbs li:last-child:not(:first-child) {
```

Comment:
Was there a test URL example for this one?

Reply:
We don't yet have any RTL locales ported over to the FEDS Gnav yet, I test by changing the attribute manually for the moment. But to your point, we always check RTL locales when making changes.